gabbi/tests/gabbits_intercept/utf8.txt

```diff
@@ -0,0 +1,222 @@
+<h1>Unicode Demo</h1>
+
+<p>Taken from <a
+href="http://www.cl.cam.ac.uk/~mgk25/ucs/examples/UTF-8-demo.txt">http://www.cl.cam.ac.uk/~mgk25/ucs/examples/UTF-8-demo.txt</a></p>
+
+<pre>
+
+$ENVIRON['HOME']
```

Comment:
Yes, that makes sense. Compare as text when we have text, and binary if not.

Ah right, replacement in data files is indeed a documented feature.